packages/client/src/pages/RecommendationsPage/RecommendationsTable/recommendationsTableStyles.ts

```diff
@@ -23,6 +23,13 @@ const useStyles = makeStyles(() => ({
   toggleContainer: {
     alignSelf: 'flex-end',
   },
+  formControl: {
+    margin: '2px',
+    minWidth: 80,
+  },
```

Comment:
The spacing between both the top and bottom of the pagination as well as between the pageSize selection and the arrows seem to be a little squished. Would you mind adding additional margin to both?